lesson02/assignment-alt/script.js

```diff
@@ -0,0 +1,83 @@
+
+// JSON DATA
+
+const profileJSON = [
+  {
+    "name": "Paolo Maldini",
+    "jobTitle": "Front End Developer",
+    "company": "Web Weavers",
+    "experience": "3 years",
+    "school": "University of Washington",
+    "major": "Marketing",
+    "email": "paolo@example.com",
+    "linkedInUrl": "paolo.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "CSS", "JavaScript", "Node", "Express"
+    ]
+  },
+  {
+    "name": "Barbara Bonansea",
+    "jobTitle": "Software Engineer",
+    "company": "Excellence in the Cloud",
+    "experience": "12 years",
+    "school": "University of Southern California",
+    "major": "Computer Science",
+    "email": "barbara@example.com",
+    "linkedInUrl": "barbara.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "JavaScript", "C", "Go"
+    ]
+  },
+  {
+    "name": "Javier Hernandez",
+    "jobTitle": "User Experience Engineer",
+    "company": "Web Sites and More",
+    "experience": "5 years",
+    "school": "Seattle University",
+    "major": "Performing Arts",
+    "email": "javier@example.com",
+    "linkedInUrl": "javier.linkedinprofile.com",
+    "codeLanguages": [
+        "HTML", "CSS"
+    ]
+  },
+  {
+    "name": "Maribel Dominguez",
+    "jobTitle": "Front End Engineer",
+    "company": "Bits and Bytes",
+    "experience": "6 years",
+    "school": "University of Washington",
+    "major": "Mechanical Engineering",
+    "email": "maribel@example.com",
+    "linkedInUrl": "maribel.linkedinprofile.com",
+    "codeLanguages": [
+       "HTML", "CSS", "JavaScript", "React", "Vue", "Redux"
+    ]
+  }
+];
+
+// Convert JSON to object
+const profileString = JSON.stringify(profileJSON);
+const profileInfoBank = JSON.parse(profileString);
+
+let output = "";
+
+// Creates Individual Cards
+profileInfoBank.forEach(function(currentUser){
+    //Creates Profile Base Div
+    let profileBase = `<div class ="card-container"><div class ="profile-base"><img src="img/headshot.jpg" alt="Dashing portrait of ${currentUser.name}"><h2>${currentUser.name}</h2><p>${currentUser.jobTitle}</p></div>`;
+
+    //Creates Profile Deets Div
+    let profileDeets = `<div class="profile-deets">`;
+
+    let jobDeets = `<div class ="company"><h3>Company: </h3><p>${currentUser.company}</p></div>`;
```

Comment:
You could actually combine most of this code, no need to separate them into variables. It is ok to have one big block of code.